Makefile

```diff
@@ -0,0 +1,40 @@
+# define the name of the virtual environment directory
+VENV := venv
+
+# default target, when make executed without arguments
+all: venv
+
+install: FORCE
```

Comment:
What is the purpose of `FORCE`

Reply:
It means that even if the dependencies haven't changed, `make force` will make it run. It circumvents the lazy/incremental running of make instructions